explainer.md

```diff
@@ -25,24 +25,40 @@ The following web application declares in its manifest that it can handle CSV an
         {
           "action": "/open-csv",
           "name": "Raw Graph",
-          "accept": [ ".csv", "text/csv" ]
-        }
+          "accept": {
+            "text/csv": [ ".csv" ]
+          }
+        },
         {
           "action": "/open-svg",
           "name": "SVG Graph",
-          "accept": [ ".svg", "text/svg+xml" ]
+          "accept": {
+            "image/svg+xml": ".svg",
+          }
+        },
+        {
+          "action": "/open-graf",
+          "name": "Grafr File",
+          "accept": {
+            "application/vnd.grafr.graph": [
+              ".grafr", ".graf"
+            ],
+            "application/vnd.alternative-graph-app.graph": [
+              ".graph",
+            ]
+          }
         }
       ]
     }
 ```
 
 > Note: `action` must be inside the app scope.
 
-Each accept entry is a sequence of MIME types and/or file extensions.
+Each accept entry is a dictionary mapping mimetypes to extensions. This ensures applications will still work on operating systems that only support one of the two.
 
-On platforms that only use file extensions to describe file types, user agents can match on the extensions ".csv" and ".svg".
+On platforms that only use file extensions to describe file types, user agents can match on the extensions ".csv", ".svg", ".grafr", ".graf" and ".graph".
 
-On a system that does not use file extensions but associates files with MIME types, user agents can match on the "text/csv" and "image/svg+xml" MIME types. If the web application accepts all text and image formats that the browser supports, "text/\*" and "image/\*" could be used, i.e. "\*" may appear in place of a subtype. "\*/\*" can be used if all files are accepted.
+On a system that does not use file extensions but associates files with MIME types, user agents can match on the "text/csv", "image/svg+xml", "application/vnd.grafr.graph" and "application/vnd.alternative-graph-app.graph" MIME types. If the web application accepts all text and image formats that the browser supports, "text/\*" and "image/\*" could be used, i.e. "\*" may appear in place of a subtype. "\*/\*" can be used if all files are accepted.
```

Comment:
nit: Swap these paragraphs (this and the one above), since MIME types are at the forefront now, and extensions are just a "fallback" for OSes that don't do MIME types.

Reply:
I think you added the .grafr example to demonstrate that you can add bespoke MIME types / file extensions which get registered (even on platforms like Linux that are based on MIME types and not file extensions). Maybe call that out.